migration/src/main/java/com/github/gfx/android/orma/migration/ManualStepMigration.java

```diff
@@ -170,11 +177,11 @@ public void run() {
             }
         }
         runTasksInTransaction(db, tasks);
+        saveStep(db, newVersion, MIGRATION_COMPLETED);
```

Comment:
This is the fix.

It means to always update DB version (i.e. `db.setVersion`) via `saveStep`.